src/plugins/query_enhancements/public/query_assist/utils/constant.ts

```diff
@@ -11,3 +11,5 @@ export const PPL_SUPPORT_DATASET_TYPES = [
   DEFAULT_DATA.SET_TYPES.INDEX,
   DEFAULT_DATA.SET_TYPES.INDEX_PATTERN,
 ];
+
+export const ENABLE_AI_FEATURES = 'enableAIFeatures';
```

Comment:
Shall we reuse the constant? I saw this value in `src/core/server/ui_settings/settings/ai_features.ts`

Reply:
Sure, but is it a good convention to reuse server constant in client-side public code?

Reply:
We can put it inside core/util and expose it to public / server side from core/public and core/server.

Reply:
Done